src/jacowvalidator/app.py

```diff
@@ -91,7 +94,7 @@ def upload():
                 if p.text.strip().lower() == 'references':
                     references_start = i
 
-            author_paragraphs = doc.paragraphs[1 : abstract['start']]
+            author_paragraphs = doc.paragraphs[1: abstract['start']]
```

Comment:
This is something i don't understand about the document structure, why are we skipping the first paragraph?

Reply:
my change to that file was just for pep8.  
First paragraph is the title.  It will not be part of the author paragraphs.